content/applications/services/helpdesk/overview/getting_started.rst

```diff
@@ -140,25 +173,30 @@ Assign stages to a team
 -----------------------
 
 Make a selection in the :guilabel:`Teams` field on the :guilabel:`Stages` form. More than one team
-may be selected, since the same stage(s) can be assigned to multiple teams.
-
- .. image:: getting_started/stages-settings-sharing.png
-   :align: center
-   :alt: View of stage setup emphasizing teams field
+may be selected, since the same stage can be assigned to multiple teams.
 
 Fold a stage
 ------------
 
-Check the :guilabel:`Folded in Kanban` box on the :guilabel:`Stages` form to display this stage as
-*folded* by default in the kanban view for this team.
+By default, stages are unfolded in kanban view. Tickets in an unfolded stage are visible in the
+pipeline under the stage name, and are considered *open*. However, stages can be configured to be
+folded in kanban. Folded stages are still visible in kanban, though the tickets in the stage are no
+longer immediately visible.
 
 .. warning::
    Tickets that reach a *folded* stage are considered closed. Closing a ticket before the work is
    completed can result in reporting and communication issues. This setting should only be enabled
    for stages that are considered *closing* stages.
 
-Alternatively, stages can be temporarily folded in the kanban view, by clicking on the settings
-icon and selecting :guilabel:`Fold`.
+To fold a stage, check the :guilabel:`Folded in Kanban` box on the :guilabel:`Stages` form.
+
+Stages can be temporarily folded in the kanban view, by clicking on the settings icon and selecting
+:guilabel:`Fold`.
 
-.. note::
-   Manually folding a stage from the kanban view will not close the tickets in the stage.
+.. image:: getting_started/fold-stage-kanban.png
+   :align: center
+   :alt: Kanban view of a Helpdesk stage, with the temporary fold option emphasized.
+
+.. important::
+   Manually folding a stage from the kanban view is temporary and will **not** close the tickets in
+   the stage.
```

Comment:
```suggestion
   the stage. Tickets are **only** automatically closed when the :guilabel:`Folded in Kanban` option is enabled.
```

Reply:
This is incorrect, this is not how tickets are closed. I do not want to add any more information than necessary here because I want to keep this document to just the basics about setup and configuration, but this is not the only way tickets can be automatically closed